samples/extensions/descriptor_buffer_basic/descriptor_buffer_basic.cpp

```diff
@@ -196,6 +201,10 @@ void DescriptorBufferBasic::setup_descriptor_set_layout()
 	vkGetDescriptorSetLayoutSizeEXT(get_device().get_handle(), uniform_binding_descriptor.layout, &uniform_binding_descriptor.size);
 	vkGetDescriptorSetLayoutSizeEXT(get_device().get_handle(), image_binding_descriptor.layout, &image_binding_descriptor.size);
 
+	// Adjust set layout sizes to alignment.
+	uniform_binding_descriptor.size = aligned_size(uniform_binding_descriptor.size, descriptor_buffer_properties.descriptorBufferOffsetAlignment);
```

Comment:
Just a comment to explain. Before this PR we had this code:
```
uniform_descriptor_offset = aligned_size(descriptor_buffer_properties.uniformBufferDescriptorSize, descriptor_buffer_properties.descriptorBufferOffsetAlignment);
	image_descriptor_offset   = aligned_size(descriptor_buffer_properties.combinedImageSamplerDescriptorSize, descriptor_buffer_properties.descriptorBufferOffsetAlignment);
```
And it was adjusting only the descriptor binding size. Then I removed it, as for the offset calculation entire set layout size should be used. But then while reading adoc and `descriptorBufferOffsetAlignment` I understood that set layout size should still be adjusted to comply with an alignment requirement.